api/database/seeders/DepartmentSeeder.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Department;
+use Illuminate\Database\Seeder;
+
+class DepartmentSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+      $departments = [
```

Comment:
Any reason why not all of the data is being seeded from the [department csv](http://donnees-data.tpsgc-pwgsc.gc.ca/ba1/min-dept/min-dept.csv)?

Reply:
I made another issue for that #486